polaris/experimental/_dataset_v2.py

```diff
@@ -117,12 +127,32 @@ def zarr_manifest_path(self) -> str:
 
         return self._zarr_md5sum_manifest_path
 
-    def _compute_checksum(self) -> str:
-        """Compute the checksum of the Zarr manifest file."""
-        manifest_md5 = calculate_file_md5(self.zarr_manifest_path)
-        return manifest_md5
+    @computed_field
+    @property
+    def md5sum(self) -> str:
```

Comment:
Why'd we move this out of the ChecksumMixin class?

Reply:
Good question!

`DatasetV2` no longer bases the `ChecksumMixin`. Since the `md5sum` is not deterministic, it is used differently than the checksum for V1. E.g. the `verify_checksum()` method doesn't make much sense. To make that difference in use explicit, I added separate attributes here. See https://github.com/polaris-hub/polaris/pull/186#discussion_r1744684996 for context.

If this is causing confusion, perhaps we could rename the attribute here to make clear it's something else. What about `manifest_md5sum` ?

Reply:
Makes sense, very nice. I think `manifest_md5sum` is a clearer name too!

Reply:
Renamed the variables!